installer/dev/README.md

```diff
@@ -6,6 +6,6 @@ packages be built. Typically this only needs to be done once unless the set of p
 To build the test packages, build CreateInstallerTestPackages.sln, located under
 installer\test\CreateInstallerTestPackages.
 
-For building an installer with real packages, create a 'project_reunion_definitions_override.h'
+For building an installer with real packages, create a 'windowsappsdk_definitions_override.h'
```

Comment:
Also consistent with other references, so it looks like just the .gitignore was inconsistent in the naming.